keras_core/utils/file_utils.py

```diff
@@ -386,10 +386,21 @@ def validate_file(fpath, file_hash, algorithm="auto", chunk_size=65535):
 
 
 def is_remote_path(filepath):
-    """Returns `True` for paths that represent a remote GCS location."""
-    # TODO: improve generality.
-    if re.match(r"^(/cns|/cfs|/gcs|.*://).*$", str(filepath)):
-        return True
+    filepath_str = str(filepath).strip()
```

Comment:
Please keep the original implementation with support for `cfs` and `cns` (you can add `hdfs` is the list of patterns). We should also not print a warning.